package.json

```diff
@@ -35,6 +35,7 @@
     "commitlint-cli": "1.1.3",
     "eslint": "8.57.0",
     "eslint-config-prettier": "9.1.0",
+    "eslint-import-resolver-node": "^0.3.9",
```

Comment:
I think I could not find where this dependency is used. Or does it work auto-magically? 🪄

Reply:
Yes, this works automagically with ESLint. The root of the problem was that the ESLint could not resolve `node:assert`. It does not know the correct namespace. So by installing this package, ESLint can resolve the `node:assert` as a part of Node.js namespace.